mlem/cli/apply.py

```diff
@@ -92,6 +99,7 @@ def apply(
                 data_rev,
                 load_value=True,
```

Comment:
as per comments in `metadata.py` and `objects.py`, this should be `load_meta(..., load_value=batch is None)`